### PR TITLE
Freewheel-SSP Bid Adapter: add alias, update local param, and other additions

### PR DESCRIPTION
**Pull request by xwang202**
## Type of change
- [ ] Feature

## Description of change
1. Added alias `freewheelssp`
2. Added the method to get the `advertiserDomains`
3. Added `iframeEnabled` option in `getUserSyncs` method

### CODE REVIEW

**Comment by xwang202** [comment]
@patmmccann can you please review? Thanks.

File: modules/freewheel-sspBidAdapter.js [hunk 1]
```diff
@@ -326,8 +358,8 @@ export const spec = {
           }
         }
       }
-      // TODO: is 'page' the right value here?
-      var location = bidderRequest?.refererInfo?.page;
+
+      var location = (bidderRequest && bidderRequest.refererInfo) ? bidderRequest.refererInfo.referer : getTopMostWindow().location.href;
```
Comment (patmmccann) [inline modules/freewheel-sspBidAdapter.js:362]: What's wrong with page? This appears incorrect

File: test/spec/modules/freewheel-sspBidAdapter_spec.js [hunk 2]
```diff
@@ -387,7 +387,7 @@ describe('freewheelSSP BidAdapter Test', () => {
       ];
 
       let result = spec.interpretResponse(response, request[0]);
-      expect(result[0].meta.advertiserDomains).to.deep.equal([]);
+      expect(result[0].meta.advertiserDomains).to.deep.equal('');
```
Comment (patmmccann) [inline test/spec/modules/freewheel-sspBidAdapter_spec.js:390]: This must be an array

**Review by patmmccann — CHANGES_REQUESTED** [review]
Also please re-title your pr to match format of recently merged prs

File: test/spec/modules/freewheel-sspBidAdapter_spec.js (hunk 2 above)
Comment (xwang202) [reply test/spec/modules/freewheel-sspBidAdapter_spec.js]: Updated.

**Comment by xwang202** [comment]
@patmmccann the code has been updated. Can you please double check? Thanks.

**Pull request merged**